README.md

```diff
@@ -4,18 +4,15 @@
 
 A recipe app for the command line that takes a text file and returns a grocery list.
 
-## Installation
+### Installation
+Download or clone this repository with the "clone or download" button at the top of this page.
 
-1. Clone this repo by clicking the green button at the top right.
+This project was built in Elixir and uses mix to start the game and run the accompanying test suite. To run this application you'll first need to install Elixir (which comes with mix) from this page here. Once installed you can run the commands detailed below from the project's root directory.
```

Comment:
Did you intend to have a link to an Elixir download here?

Reply:
Sure did. Updating! 🔗